### ISSUE TITLE
Multiple CalDAV accounts on same server

### ISSUE THREAD

**Issue by skalarproduktraum** [marcoancona/MineTime#241]
It seems like adding multiple accounts residing on the same server (with different URL endpoints) is not possible if the username is the same.

**Comment by marcoancona** [comment]
Right, this is a scenario we did not consider. At the moment, we use the pair (username, hostname) as an identifier for an account. The use of hostname is justified by the fact that normally you want to connect an account by entering only the domain and leave the resolution of the entry points to the autodiscovery, which would not work in case multiple servers are on the same hostname.
